<launch>

    <node pkg="guiding_as" type="show_as.py" name="show_action_server" output="screen"/>
    <rosparam command="load" file="$(find guiding_as)/params/show.yaml" />

</launch>
